python/tvm/driver/tvmc/model.py

```diff
@@ -0,0 +1,331 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+The TVMC Model class and helper functions.
```

Comment:
Some high level documentation would be useful here. Also example usage would be helpful. Maybe you can copy the example from the tests?

Reply:
I improved this docstring, does this look better now?

Reply:
yes this looks better. I'm not sure if the markdown formatting will render correctly. I think you might have to use rtf.